src/BugsnagUnity/Client.cs

```diff
@@ -95,6 +97,15 @@ public Client(INativeClient nativeClient)
             }
         }
 
+        private void SetupSceneLoadedBreadcrumbTracking()
+        {
+            if (Configuration.EnabledBreadcrumbTypes == null ||
+               Configuration.EnabledBreadcrumbTypes.Contains(BreadcrumbType.Navigation))
```

Comment:
Since the logic combo `types == null or types.contains(x)` is used a few places, it would make sense to convert it to a function, maybe on `Configuration`?

```suggestion
            if (Configuration.IsBreadcrumbTypeEnabled(BreadcrumbType.Navigation))
```

Reply:
done